Docs/GettingStarted.md

```diff
@@ -49,14 +49,13 @@ This will build and run the Foundation tests, in the Debug configuration.
 After the complete Swift build has finished, you can iterate quickly on Foundation itself by simply invoking `ninja` in the Foundation directory.
 
 ```
-cd swift-corelibs-foundation
-ninja
+ninja -C build/Ninja-DebugAsserts/swift-corelibs-foundation Foundation
 ```
 
 This will build Foundation. To build and run the tests, use the `test` target:
 
 ```
-ninja test
+ninja -C build/Ninja-DebugAsserts/swift-corelibs-foundation test
```

Comment:
That doesn't seem to work:

```$ ninja -C build/Ninja-ReleaseAssert/foundation-linux-x86_64  test
ninja: Entering directory `build/Ninja-ReleaseAssert/foundation-linux-x86_64'
[0/1] Running tests...
Test project /home/spse/swift-source/build/Ninja-ReleaseAssert/foundation-linux-x86_64
No tests were found!!!
```

Reply:
@spevans - did you enable the tests?  `grep ENABLE_TESTING build/Ninja-ReleaseAssert/foundation-linux-x86_64/CMakeCache.txt`

Reply:
This _DEFINITELY_ must make into the documentation if it isn't there. It's not obvious.

Reply:
```
$ grep ENABLE_TESTING build/Ninja-ReleaseAssert/foundation-linux-x86_64/CMakeCache.txt 
ENABLE_TESTING:BOOL=NO
```

How are the tests enabled? Is it only if `--test-foundation` is passed to `build-script-impl`?
I don't really know much about CMake but could the tests always be enabled by `build-script-impl` even it `--test-xxx` flags aren't passed? I take it actually running the tests is still a separate step in the build script?

Reply:
I'm confused, why are tests disabled by default?

Reply:
because testing enables support for `@testable import`, which we don't want to have on in the release build.